docs/grid.html

```diff
@@ -0,0 +1,270 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Grid</title>
+    <link rel="stylesheet" href="../components/grid/grid.css">
+    <meta name="theme-color" content="#2C2929">
+
+</head>
+<body>
+    <nav class="nav z-index-max">
+        <div class="flex flex-row nav-logo flex-between flex-align-item-center ">
+            <div class="sidebar-logo flex flex-align-item-center">
+                <span class="material-icons icon-nav-menu" id="menu">
+                    menu
+                </span>
+                <span class="logo-text">Framer-Ui</span>
+            </div>
+            <div class="flex flex-wrap gap">
+                <a href="https://github.com/mehulsatardekar"  target="_blank" rel="noreferrer">
+                    <box-icon name='github' type='logo' color='#eeeded' ></box-icon>  
+                </a>  
+                <a href="https://www.youtube.com/channel/UCOPdSGIn44Gh4QXZH_6U8gQ"  target="_blank" rel="noreferrer">
+                    <box-icon name='youtube' type='logo' color='#eeeded' ></box-icon>  
+                </a> 
```

Comment:
Suggestion: I am not sure but you can use css variables here

Reply:
> Suggestion: I am not sure but you can use css variables here

It's specific to the icons which I am importing from (boxicons).